packages/node-app/src/tests/location.controller.test.ts

```diff
@@ -1,8 +1,20 @@
-import request from 'supertest';
 import mongoose from 'mongoose';
+import request from 'supertest';
+
 import { app } from '../app';
+import { verifyToken } from '../middlewares/auth.middleware';
 import { Location } from '../models/location.model';
 
+jest.mock('../middlewares/auth.middleware');
+const mockedVerifyToken = jest.mocked(verifyToken);
+
+const authenticated = (testFn) => {
```

Comment:
Let's not duplicate code. Instead we can create a `test.utils` file and import this from there